### PR TITLE
Remove Welcome modal in popup and other bug fixes

### PR DESCRIPTION
**Pull request by mindofmar**
- [x] Removes posthog feature flags & welcome modal since we don't really need to show this promo any longer
- [x] Adds same validation for bypass as what is in the normal content script
- [x] Fixes issue where some DApps set the `personal_sign` args backwards, which can cause issues in our data

### CODE REVIEW

File: src/injected/injectWalletGuard.tsx [hunk 1]
```diff
@@ -427,8 +434,15 @@ const addWalletGuardProxy = (provider: any) => {
           return Reflect.apply(target, thisArg, args);
         }
 
-        const signer: string = request.params[1];
-        const signMessage: string = request.params[0];
+        let signer: string = request.params[1];
+        let signMessage: string = request.params[0];
+
+        // Some DApps send these params in reverse
+        if (signer.substring(0, 2) !== '0x' && signMessage.substring(0, 2) === '0x') {
```
Comment (AugmentedMode) [inline src/injected/injectWalletGuard.tsx]: should this be a function as you use the same code twice in this file

File: src/content-scripts/bypassCheck.tsx [hunk 2]
```diff
@@ -46,45 +47,71 @@ window.addEventListener('message', (message) => {
...
         return;
       }
 
-      const signer: string = data.params[1];
-      const signMessage: string = data.params[0];
+      let signer: string = data.params[1];
+      let signMessage: string = data.params[0];
+
+      // Some DApps send these params in reverse
+      if (signer.substring(0, 2) !== '0x' && signMessage.substring(0, 2) === '0x') {
```
Comment (AugmentedMode) [inline src/content-scripts/bypassCheck.tsx]: looks like you use the same code here as well

**Review by AugmentedMode — APPROVED** [review]

**Review by AugmentedMode — DISMISSED** [review]
Did you test the bypass functionality change?

File: src/injected/injectWalletGuard.tsx (hunk 1 above)
Comment (mindofmar) [reply src/injected/injectWalletGuard.tsx]: refactored this into a function, good call

**Comment by mindofmar** [comment]
> Did you test the bypass functionality change?

yeah, seems to be working fine on revoke extension tests

**Review by AugmentedMode — APPROVED** [review]

**Pull request merged**